### PR TITLE
Adds XBOX controller for velocity input mode

### PR DESCRIPTION
**Pull request by aftersomemath**
Also turns robot localization update rate up to 100 Hz.

### CODE REVIEW

File: launch/motion_stack.launch [hunk 1]
```diff
@@ -11,10 +11,12 @@
         <arg name="bond_id_namespace" value="$(arg bond_id_namespace)" />
     </include>
 
+<!--
     <include if="$(eval str(arg('platform')) != '1.5')"
         file="$(find iarc7_planner)/launch/planner.launch">
         <arg name="platform" value="$(arg platform)" />
         <arg name="bond_id_namespace" value="$(arg bond_id_namespace)" />
     </include>
+-->
```
Comment (asaba96) [inline launch/motion_stack.launch:20]: Why are we commenting this out?

File: launch/base_hardware.launch [hunk 2]
```diff
@@ -95,9 +95,16 @@
...
 
+        <node pkg="joy" type="joy_node" name="joy_node">
+            <param name="deadzone" value="0.1" />
+            <param name="autorepeat_rate" value="50" />
+            <param name="coalesce_interval" value="0.02" />
+        </node>
 
         <include file="$(find iarc7_sensors)/launch/nano_subscriber.launch" />
 
+        <include file="$(find iarc7_sensors)/launch/flow_transformer.launch" />
```
Comment (asaba96) [inline launch/base_hardware.launch:106]: Is the code for this node merged?

File: params/robot_localization_config.yaml [hunk 3]
```diff
@@ -1,5 +1,5 @@
 # The frequency, in Hz, at which the filter will output a position estimate
-frequency: 30
+frequency: 100
```
Comment (asaba96) [inline params/robot_localization_config.yaml:2]: Why increase the frequency? Is there any degrade in performance?

File: launch/base_hardware.launch (hunk 2 above)
Comment (asaba96) [reply launch/base_hardware.launch:106]: If not, I don’t want to merge this so to avoid breaking master.

File: launch/motion_stack.launch (hunk 1 above)
Comment (aftersomemath) [reply launch/motion_stack.launch:20]: Because for almost everyone it doesn't work.

File: params/robot_localization_config.yaml (hunk 3 above)
Comment (aftersomemath) [reply params/robot_localization_config.yaml:2]: Increasing the frequency does not degrade performance.

Comment (asaba96) [reply params/robot_localization_config.yaml:2]: I mean CPU performance. Like is the jetson able to keep up?

Comment (aftersomemath) [reply params/robot_localization_config.yaml:2]: yes

Comment (aftersomemath) [reply params/robot_localization_config.yaml:2]: this was all flight tested.

File: launch/motion_stack.launch (hunk 1 above)
Comment (amiller27) [reply launch/motion_stack.launch:20]: Yeah I shouldn't have merged that

File: launch/base_hardware.launch [hunk 4]
```diff
@@ -95,9 +95,16 @@
             <param name="baud" value="115200" />
         </node>
 
+        <node pkg="joy" type="joy_node" name="joy_node">
```
Comment (amiller27) [inline launch/base_hardware.launch:98]: Can't be launching this by default

**Review by amiller27 — COMMENTED** [review]
Looks good except for that one thing

File: launch/base_hardware.launch (hunk 4 above)
Comment (aftersomemath) [reply launch/base_hardware.launch:98]: As discussed this can be launched by default since its a hardware component.

**Review by amiller27 — APPROVED** [review]

**Pull request merged**